### PR TITLE
Style fixes and minor typos.

### PR DESCRIPTION
**Pull request by panchoh**

### CODE REVIEW

File: spec/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Because we think that:
 
 - UNIX® is great and we love it
 - The UNIX® shell is great
-- HTTP interfaces are convenient and everywhere
+- HTTP interfaces are convenient and ubiquitous
```
Comment (nilp0inter) [inline spec/README.md]: -0, not [Basic English](https://simple.wikipedia.org/wiki/Basic_English)

File: spec/README.md [hunk 2]
```diff
@@ -58,8 +58,8 @@ Settings through two command line arguments, path and shell command.
 
 Tools with a rigid matching **can't evade** *[impedance
 mismatch](https://haacked.com/archive/2004/06/15/impedance-mismatch.aspx/)*.
-Resulting in an easy-to-use software, convenient in some scenarios but
-incapable in others.
+Resulting in software that is convenient and easy-to-use in a handful of scenarios,
+but sorely lacking in the vast majority of them.
```
Comment (nilp0inter) [inline spec/README.md]: Do not express original intent. Also not Basic English.

File: spec/README.md (hunk 1 above)
Comment (panchoh) [reply spec/README.md]: Fair enough.  Arigato!

Comment (panchoh) [reply spec/README.md]: (reverted)

File: spec/README.md (hunk 2 above)
Comment (panchoh) [reply spec/README.md]: (reverted).  Merci !

**Pull request merged**